release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -717,6 +718,15 @@ With this release, the NUMA Resources Operator no longer creates a custom SELinu
 In clusters with an existing NUMA-aware scheduler configuration, upgrading to {product-title} 4.18 might result in an additional reboot for each configured node. For further information about how to manage an upgrade in this scenario and limit disruption, see the Red Hat Knowledgebase article link:https://access.redhat.com/articles/7107603[Managing an upgrade to {product-title} 4.18 or later for a cluster with an existing NUMA-aware scheduler configuration]
 ====
 
+[id="ocp-release-notes-scalability-and-performance-nto-platforms_{context}"]
+==== Node Tuning Operator platform detection
+
+With this release, when you apply a performance profile, the Node Tuning Operator detects the platform and configures kernel arguments and other platform-specific options accordingly. This release adds support for detecting the following platforms:
+
+* AMD x86
+* ARM aarch64
+* Intel x86
```

Comment:
The RH Supp guide, states this architectures as:

* AMD64
* AArch64
* Intel 64

Reply:
https://redhat-documentation.github.io/supplementary-style-guide/#aarch64

Reply:
Will a link exists to the feature docs? Not sure if https://88630--ocpdocs-pr.netlify.app/openshift-enterprise/latest/scalability_and_performance/cnf-tuning-low-latency-nodes-with-perf-profile might work?

Reply:
Updated the architectures. Thanks! There's no real customer-facing docs changes to point to. The NTO is installed by default. I think don't need to link to NTO, it's just flagging the change how it works mostly behind the scenes